aptos-move/vm-genesis/src/lib.rs

```diff
@@ -225,7 +225,7 @@ fn create_and_initialize_main_accounts(
         bcs::to_bytes(&consensus_config).expect("Failure serializing genesis consensus config");
 
     // TODO: Make reward rate configurable in the genesis blob.
-    let rewards_rate_percentage = 1;
+    let rewards_rate = 10; // 0.001% per epoch or ~10% APY if 1 epoch = 1 hour.
```

Comment:
does the epoch length exist here? ideally we should derive the number instead of hardcode it